freeform_json_to_yaml.py

```diff
@@ -0,0 +1,440 @@
+                'ambulatory': c['triageStatus']['sort'] == 'walker',
+                'mental_status': MENTAL_STATUS_MAP[vitals['mood']],
+                'breathing': BREATHING_MAP[vitals['breath']],
+                'heart_rate': PULSE_MAP[vitals['pulse']],
+                'spo2': 67 if vitals['SpO2'] == 'none' else 85 if vitals['SpO2'] == 'low' else 95
+            }
+            injuries = []
+            written_injuries = {}
+            for i in c['injuries']:
+                if i['type'] == 'Asthmatic':
+                    injury = {
+                        'name': 'Asthmatic',
+                        'location': "internal",
+                        'status': 'hidden'
+                    }
+                    written_injuries[injury['name']] = ['internal']
+                elif i['type'] == 'Ear Bleed':
+                    injury = {
+                        'name': 'Ear Bleed',
+                        'location': "right face" if randint(1, 2) > 1 else "left face",
+                        'status': 'discoverable'
+                    }
+                    written_injuries[injury['name']] = [injury['location']]
+                else:
+                    if i['type'] == 'Forehead Scrape':
+                        injury = {
+                            'name': "Laceration",
+                            'location': "left face",
+                            'status': 'discoverable'
+                        } 
+                    else:
+                        split_name = i['type'].split(' ')
+                        injury = {
+                            'name': split_name[-1] if 'Broken' not in i['type'] else 'Broken Bone',
+                            'location': INJ_LOC_MAP[(' ').join(split_name[:-1])],
+                            'status': 'discoverable'
+                        }
+                    bloodPool = i.get('bloodPool', None)
+                    if bloodPool is not None:
+                        severity = SEVERITY_MAP[bloodPool]
+                    elif 'Burn' in i['type']:
+                        severity = 'major'
+
+                    injury['severity'] = severity
+                    
+                    if injury['name'] in written_injuries:
+                        written_injuries[injury['name']].append(injury['location'])
+                    else:
+                        written_injuries[injury['name']] = [injury['location']]
+                injuries.append(injury)
+            new_c['injuries'] = injuries
+            if len(written_injuries) == 0:
+                unstructured_inj = 'Has no known injuries.'
+            else:
+                unstructured_inj = "Has"
+                pronoun = 'his' if demographics['sex'] == 'M' else 'her'
+                is_asthmatic = False
+                for inj_set in written_injuries:
+                    if len(written_injuries[inj_set]) == 1:
+                        if inj_set == 'Collapse':
```

Comment:
This isn't an injury type in the API, and is causing a validation error in the YAML generated from the Metro JSONs.  Doesn't it need to be "Chest Collapse"?